cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -300,6 +275,11 @@ def report_number(self, key, value):
 @model.over("identifiers", "^970__")
 @for_each_value
 def aleph_number(self, key, value):
+    """Translates identifiers: ALEPH.
+
+        Attention:  035 might contain aleph number
+        https://github.com/CERNDocumentServer/cds-migrator-kit/issues/21
```

Comment:
Aleph n. should also be added as an alternative identifier, right?

Reply:
yes, and it is already done.
The comment there is pointing to other collections which have aleph in 035 (as of investigation from Ludmila) so I left it there as a reminder to check for the future